todo_app.js

```diff
@@ -1,10 +1,108 @@
 
+
+function TodoApp() {
+    //your code to run since DOM is loaded and ready;
+    document.addEventListener("DOMContentLoaded", TodoApp.prototype = function (event) {
```

Comment:
noooooooooooooooooooo....

You should move app creation logic from constructor. Create app in event handler.